src/sardana/pool/test/base.py

```diff
@@ -171,13 +175,13 @@ def setUp(self):
                 name = '_test_tg_%s_%s' % (ctrl, axis)
                 self.createTGElement(ctrl_obj, name, axis)
         # Create nctrls MOT ctrls
-        for ctrl in range(1, self.nctctrls + 1):
+        for ctrl in range(1, self.nmotctrls + 1):
             name = '_test_mot_ctrl_%s' % ctrl
             ctrl_obj = self.createController(name,
                                              'DummyMotorController',
                                              'DummyMotorController.py')
             # Create nelems CT elements for each ctrl
-            for axis in range(1, self.nctelems + 1):
+            for axis in range(1, self.nmotelems + 1):
```

Comment:
Hi, I was analysing the changes and this one surprised me. Did you change `self.nctctrls` by `self.nmotelems` intentionally **in this case**?
I see a change before where the same change makes sense according to the comment above, but in this case the comment refers to "CT elements"

Reply:
Thanks! The code is correct. Te comment also needs to be changed to "create nelems MOT elements...".
And the one above regarding the trigger gate element needs to be changed similarly.

Reply:
ok. I'll do the changes and merge then